### DISCUSSION TITLE
Bun JS support

### DISCUSSION THREAD

**Discussion by kronos-999** [uNetworking/uWebSockets.js#759] [Ideas]
Recently, a new and fast JS runtime was released. It is written is Zig and looks promising, I didn't look deeper into it though. Might wanna look at it as it matures ;)

**Comment by mikrosystheme** [comment]
Bun uses uWS already, so you have nothing to support.

**Comment by FlorianBELLAZOUZ** [comment]
You can see more information here
https://github.com/uNetworking/uWebSockets/discussions/1466#discussioncomment-3114410

**Reply by kronos-999** [reply]
Thanks, didn't see this dicussion
